src/Configuration/PlatesFormatterConfiguration.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Equip\Configuration;
+
+use Auryn\Injector;
+use Equip\ContentNegotiation;
+use Equip\Formatter\PlatesFormatter;
+
+class PlatesFormatterConfiguration implements ConfigurationInterface
```

Comment:
Having this configuration is, more than anything, a useful example as to how to enable additional formatters within content negotiation. I am reluctant to delete our only example.

I had forgotten about the `withTemplate` problem though... we might need rethink that. It might make more sense for ContentNegotiation to be defined as middleware that sets the preferred formatter as a request attribute. Maybe also configures injection for it too...